### ISSUE TITLE
[1.9.25] iPhone 6 Plus / 10.0.2, iPhone 6S / 10.0.1 - Application is crashing when user launch the app.

### ISSUE THREAD

**Issue by testershub** [Lomotif/lomotif-ios-issues#766]
Frequency: Every time

Action:
1: Uninstall the previous version of lomotif.
2: Fresh Install and launch the app.
3: Check if application is crashing

Expected result: Application should not crash.

Actual result: Application is crashing when user launch the app.

Note: Please see issue is specifically observed on 10.0.2 and 10.0.1 OS. It is working as expected on 9.3.4 and 9.3.5 OS.

**Comment by testershub** [comment]
Refer following video for more details:
http://www.screencast.com/t/wgJtuIQA8f

**Comment by hiteshbheda** [comment]
@xiuie I can't reproduce this issue for build [1.9.25] on iPhone 6 / iOS 10.0.2

**Comment by hiteshbheda** [comment]
@xiuie Yes, I can reproduce this issue for fresh installation for build 1.9.25 on iPhone 6S / iOS 10.0.1
